### ISSUE TITLE
Accessibility requirements in one place

### ISSUE THREAD

**Issue by murata2makoto** [w3c/epub-specs#897]
EPUB Content Documents 3.1 mentions accessibility in several places, but we now have a separate specification " EPUB Accessibility ".  All accessibility requirements should be stated in this specification and nowhere else.

Thus, I propose to drop

[It should recognize embedded ARIA markup and support exposure of any given ARIA roles, states and properties to platform accessibility APIs [WAI-ARIA].](https://rawgit.com/IDPF/epub-revision/master/build/31/spec/epub-contentdocs.html#confreq-html-rs-a11y)

[Accessibility
  EPUB Content Documents that include scripting should employ relevant accessibility techniques to ensure that the content remains consumable by all users. [WAI-ARIA] [WCAG 2.0]](https://rawgit.com/IDPF/epub-revision/master/build/31/spec/epub-contentdocs.html#confreq-cd-scripted-a11y)

**Comment by mattgarrish** [comment]
1) This shouldn't be moved, as the accessibility guidelines are not about reading system implementations. I'd question the value of the statement, though, since it seems to lower what is required by html.

2) We left the WAI-ARIA requirement at the request of that group. The accessibility guidelines generally strengthen requirements of the core specs, which doesn't mean the core specs can't say anything about accessibility. There's no need to cite wcag here, so this could just read: "EPUB Content Documents that include scripting should employ relevant [WAI-ARIA] techniques to ensure that the content remains consumable by all users."

**Comment by murata2makoto** [comment]
So, do you intend to remove the first sentence?

**Comment by mattgarrish** [comment]
I need to review next week, but my inclination is to drop it because this is a stronger statement:

User agents are required to implement ARIA semantics on all HTML elements , as defined in the ARIA specifications [WAI-ARIA] and [core-aam-1.1].

https://www.w3.org/TR/html51/dom.html#user-agent-requirements

What bothers me is the statement that an RS doesn't have to support the DOM. If that's true, I'm not sure how an AT can render an EPUB at all. It certainly can't expose the ARIA semantics or meet the above requirement.

**Comment by mattgarrish** [comment]
I've been trying to make sense of this but am not getting very far.

If a reading system supports the dom, this weakens the requirement in HTML.

If a reading system doesn't support the dom, I guess this statement is either recommending to somehow/someway still make the aria semantics available, or conceding that that's not really realistic and it's okay not to.

If I were a betting man, I'd wager that this is tied into epub2 ideas of custom rendering engines, where only accessibility of the device would be whatever it self-voices.

Applied to all epub 3 reading systems, it suggests that exposing aria semantics is less important than HTML5 says it is.

Long story short, my best suggestion is that we throw this statement out. If there is lack of support by some reading systems that don't support the dom, it falls under the nebulous statement that support is not required.

Otherwise, we need to clarify that the lesser requirement only applies if the dom is not supported, but that in turn would seem to require actually spelling out what it means when we say the dom doesn't have to be supported.

**Comment by mattgarrish** [comment]
We discussed the reading system question today, and it was noted that some reading systems might not expose aria semantics even if they are supplied  (e.g., braille or audio optimized). So the decision was to leave everything as is (modulo the change above about dropping wcag as it's already covered).

To summarize:
- the "should" for ARIA techniques is less than the accessibility specification requires, and our goal was not to remove every potential accessibility statement
- there is a potential case for reading systems not to expose semantics, so we will leave the requirement as-is
